pkgs/servers/monitoring/prometheus/knot-exporter.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, lib, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "knot-exporter-unstable";
+  version = "2020-01-30";
```

Comment:
```suggestion
  pname = "knot-exporter";
  version = "unstable-2020-01-30";
```

Reply:
Even though that's pretty inconsistent, I think that `-unstable` on `pname` is preferred: #68518

Reply:
https://github.com/NixOS/nixpkgs/issues/68518#issuecomment-530555222

> - unstable variants of packages – should go to pname
> - main package but version obtained from VCS (because the latest release is too old and broken) – should go to version

Reply:
@SuperSandro2000 I see :) applied suggestion.